apps/modernization-ui/src/components/ButtonActionMenu/ButtonActionMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@trussworks/react-uswds';
+import { ReactElement, useState, useRef, useEffect } from 'react';
+import styles from './buttonActionMenu.module.scss';
+import { Icon as NBSIcon } from 'components/Icon/Icon';
+
+type Props = {
+    label: string;
+    items: ReactElement[];
+    showHide?: boolean;
```

Comment:
question:  The component is using `open` to decide when to show the menu, is the `showHide` property necessary?